src/main.cc

```diff
@@ -1027,6 +916,119 @@ mainRotate(void)
     externalAclInit();
 }
 
+/// perform legacy actions for process signal handling
+/// TODO: move actions into module runners
+class LegacySignalRr : public RegisteredRunner
```

Comment:
These actions are not necessarily "legacy"; they are simply misplaced. That is why the TODO comment says we want to _move_ them (rather than deprecate/replace -- things that usually happen to _legacy_ code). I suggest calling this class MainRr, MyRr, or something "local" like that.

I do not insist on this change.